tests-integration/tests/http.rs

```diff
@@ -192,6 +192,34 @@ pub async fn test_sql_api(store_type: StorageType) {
     assert!(body.execution_time_ms().is_some());
     assert!(body.error().unwrap().contains("not found"));
 
+    // test database given
+    let res = client
+        .get("/v1/sql?database=public&sql=select cpu, ts from demo limit 1")
```

Comment:
Maybe I miss something, is space ("` `") allowed in URL?

Reply:
Maybe the client handles url encode for us